material-ui-icons/build.boot

```diff
@@ -38,4 +38,4 @@
               :externs [#"material-ui-icons.ext.js"])
    (pom)
    (jar)
-   (validate)))
+   (validate-checksums)))
```

Comment:
`validate` is preferred to just `validate-checksums`

Reply:
oh okay, thought that validate-checksums is better to use. Will do it with validate from now :)